search/search-domain/src/main/java/io/camunda/search/entities/VariableEntity.java

```diff
@@ -17,7 +17,5 @@ public final record VariableEntity(
     String fullValue,
     boolean isPreview,
     Long scopeKey,
-    Long processInstanceKey,
-    Long processDefinitionKey,
-    String bpmnProcessId,
+    Long processInstanceId,
```

Comment:
Shouldn't it be `processInstanceKey`? See example:

<img width="438" alt="image" src="https://github.com/user-attachments/assets/4e5a9afa-4971-437f-8e82-6e8d19e58380">